lib/chromium.js

```diff
@@ -134,8 +134,10 @@ module.exports = function (b, opts) {
           pageFinish();
           return;
         }
-        output.write(sourceMapper.line(consumer, text));
-        output.write('\n');
+        if (!options.dumpio) {
```

Comment:
I'm wondering if this behaviour (nothing but the header written to stdout) should be mentioned in the docs somehow, as I found it a little surprising when reading the test?

Reply:
I added an inline comment to explain that this is here to avoid printing everything twice when the option is enabled.